siriuspy/siriuspy/clientconfigdb/types/global_config.py

```diff
@@ -628,6 +631,27 @@ def get_dict():
     ['SI-Fam:TI-BPM-PsMtn:State-Sel', 0, 0.0],
     ['SI-Fam:TI-BPM-PsMtn:DeltaDelayRaw-SP', 30*[0, ], 0],
     ['SI-Fam:TI-BPM-PsMtn:LowLvlLock-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-PsMtn:Direction-Sel', 0, 0.0],
+
+    ['SI-Fam:TI-BPM-MonitClk:DelayRaw-SP', 0, 0],
+    ['SI-Fam:TI-BPM-MonitClk:WidthRaw-SP', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:NrPulses-SP', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:Polarity-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:Src-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:State-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:DeltaDelayRaw-SP', 30*[0, ], 0],
+    ['SI-Fam:TI-BPM-MonitClk:LowLvlLock-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-MonitClk:Direction-Sel', 0, 0.0],
+
+    ['SI-Fam:TI-BPM-OrbIntlk:DelayRaw-SP', 0, 0],
+    ['SI-Fam:TI-BPM-OrbIntlk:WidthRaw-SP', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:NrPulses-SP', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:Polarity-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:Src-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:State-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:DeltaDelayRaw-SP', 30*[0, ], 0],
+    ['SI-Fam:TI-BPM-OrbIntlk:LowLvlLock-Sel', 0, 0.0],
+    ['SI-Fam:TI-BPM-OrbIntlk:Direction-Sel', 0, 0.0],
```

Comment:
Would it make sense to document Direction==1 here?

Reply:
yes, it would! I was going to do it, but I forgot. thanks!

Reply:
done!